Dockerfile

```diff
@@ -65,7 +60,15 @@ RUN mv target/$(cat rust_target.txt)/release/helium_gateway .
 FROM alpine:3.17.1
 ENV RUST_BACKTRACE=1
 ENV GW_LISTEN="0.0.0.0:1680"
-RUN apk add --no-cache --update curl
+RUN apk add --no-cache --update \
+    curl \
```

Comment:
we were not able to remove the dependency on the curl implementation to retrieve remote entropy from the entropy service yet or is this needed for something else?

Reply:
Good catch

@JayKickliter curl is no longer required